src/scripts.js

```diff
@@ -46,49 +46,11 @@ const activityNumStepsInfo = document.getElementById('activity-numSteps-input')
 const activityMinsActiveInfo = document.getElementById('activity-minsActive-input')
 const activityFlightsOfStairsInfo = document.getElementById('activity-flightsOfStairs-input')
 
-const sidebarName = document.getElementById('sidebarUserName');
-const stepGoalCard = document.getElementById('userStepGoalCard');
-const avStepGoalCard = document.getElementById('averageStepsGoalCard');
-const headerText = document.getElementById('headerText');
-const userAddress = document.getElementById('userAddress');
-const userEmail = document.getElementById('userEmail');
-const userStridelength = document.getElementById('userStridelength');
-const friendList = document.getElementById('friendList');
-const hydrationToday = document.getElementById('hydrationToday');
-const hydrationAverage = document.getElementById('hydrationAverage');
-const hydrationThisWeek = document.getElementById('hydrationThisWeek');
-const hydrationEarlierWeek = document.getElementById('hydrationEarlierWeek');
-const historicalWeek = document.querySelectorAll('.historicalWeek');
-const sleepToday = document.getElementById('sleepToday');
-const sleepQualityToday = document.getElementById('sleepQualityToday');
-const avUserSleepQuality = document.getElementById('avUserSleepQuality');
-const sleepThisWeek = document.getElementById('sleepThisWeek');
-const sleepEarlierWeek = document.getElementById('sleepEarlierWeek');
-const friendChallengeListToday = document.getElementById('friendChallengeListToday');
-const friendChallengeListHistory = document.getElementById('friendChallengeListHistory');
-const bigWinner = document.getElementById('bigWinner');
-const userStepsToday = document.getElementById('userStepsToday');
-const avgStepsToday = document.getElementById('avgStepsToday');
-const userStairsToday = document.getElementById('userStairsToday');
-const avgStairsToday = document.getElementById('avgStairsToday');
-const userMinutesToday = document.getElementById('userMinutesToday');
-const avgMinutesToday = document.getElementById('avgMinutesToday');
-const userStepsThisWeek = document.getElementById('userStepsThisWeek');
-const userStairsThisWeek = document.getElementById('userStairsThisWeek');
-const userMinutesThisWeek = document.getElementById('userMinutesThisWeek');
-const bestUserSteps = document.getElementById('bestUserSteps');
-const streakList = document.getElementById('streakList');
-const streakListMinutes = document.getElementById('streakListMinutes')
-
```

Comment:
Looked at this all together as we moved things around - feels good to clean up this chunk!